cmake/SetupCharm.cmake

```diff
@@ -26,11 +26,17 @@ endif(NOT TARGET Charmxx)
 # will call your normal compiler, set at charm++ installation time internally.
 # Note: The -pthread is necessary with Charm v6.10 to get linking working
 #       with GCC
-string(
-    REGEX REPLACE "<CMAKE_CXX_COMPILER>"
-    "${CHARM_COMPILER} -pthread -module EveryLB -no-charmrun"
-    CMAKE_CXX_LINK_EXECUTABLE "${CMAKE_CXX_LINK_EXECUTABLE}")
-
+if (USE_SCOTCH_LB)
+  string(
+      REGEX REPLACE "<CMAKE_CXX_COMPILER>"
+      "${CHARM_COMPILER} -pthread -module EveryLB -module ScotchLB -no-charmrun"
```

Comment:
Can you just append the `-module ScotchLB` flag instead of duplicating the command?